bears/documentation/DocumentationStyleBear.py

```diff
@@ -82,9 +83,13 @@ def process_documentation(self,
                 if stripped_desc[0] != '':
                     stripped_desc.insert(0, '')
 
-            # Indent with 4 spaces.
-            stripped_desc = ('' if line == '' else ' ' * indent_size
-                             + line for line in stripped_desc)
+            # Indent with 4 spaces or a tab.
+            if use_spaces:
+                stripped_desc = ('' if line == '' else ' ' * indent_size +
+                                 line for line in stripped_desc)
+            else:
+                stripped_desc = ('' if line == '' else '\t' +
+                                 line for line in stripped_desc)
```

Comment:
can you make `indent_size = 1` in the else part. Then write 
`'\t' * indent_size`